examples/step-11/step-11.cc

```diff
@@ -76,27 +76,31 @@ namespace Step11
   class LaplaceProblem
   {
   public:
-    LaplaceProblem (const unsigned int mapping_degree);
-    void run ();
+    LaplaceProblem(const unsigned int mapping_degree);
+    void
+    run();
 
   private:
-    void setup_system ();
-    void assemble_and_solve ();
-    void solve ();
-
-    Triangulation<dim>   triangulation;
-    FE_Q<dim>            fe;
-    DoFHandler<dim>      dof_handler;
-    MappingQ<dim>        mapping;
+    void
+    setup_system();
+    void
+    assemble_and_solve();
```

Comment:
We're going to have to fix this up in the tutorials. In the tutorials, we are constrained by vertical space and this doesn't look good with the separated return type because then we'd have to also add one extra empty line between successive function declarations.

Reply:
We don't have to fix this up right away, thought it may be nice to have an override in `examples/'

Reply:
Done. We don't enforce the return type to be on a separate line anymore.